static/js/eve-ui-config.js

```diff
@@ -6,7 +6,7 @@ var eveui_imageserver = function(image_ref) {
 	return eve_image(encodeURI(image_ref), 'png');
 };
 // fix for getting names for modules from ccp api that are in the users browser language
-eveui_accept_language = document.getElementById('lang-code').getAttribute('content');
+eveui_accept_language = document.getElementById('htmltag').getAttribute('lang');
```

Comment:
may be an idea to make this line:
```suggestion
eveui_accept_language = document.documentElement.getAttribute('lang');
```

Reply:
https://github.com/SpeedProg/eve-inc-waitlist/pull/12/commits/9f9443dea7c5098f7c8bea6aafe3037718fd7735 resolves this.